src/lib.rs

```diff
@@ -82,6 +82,9 @@ pub use with_const_generics::ConstGenericRingBuffer;
 #[cfg(feature = "generic-array")]
 mod with_generic_array;
 
+#[cfg(feature = "asynchronous")]
+mod with_async;
```

Comment:
Should be removed until we actually have this, as this currently breaks `cargo fmt`